special-interest-groups/sig-docs/member-list.md

```diff
@@ -47,11 +49,11 @@
 * [ericsyh](https://github.com/ericsyh)
 * [glkappe](https://github.com/glkappe)
 * [GMHDBJD](https://github.com/GMHDBJD)
+* [ireneontheway](https://github.com/ireneontheway)
```

Comment:
I think Irene is still not ready for being a reviewer.

Reply:
Updated.